### ISSUE TITLE
Find and replace would be a nice addition

### ISSUE THREAD

**Issue by grahamgamedev** [Jachdich/lime-ide#5]
find and replace should bi in the edit menu and use the conventional CTRL F shortcut.

**Comment by Jachdich** [comment]
true. Feel free to put any additions that would be good as an issue, it's a good way of keeping track. IK there's a long way to go.
